example/cumulus-tf/variables.tf

```diff
@@ -268,3 +268,9 @@ variable "es_index_shards" {
   type        = number
   default     = 2
 }
+
+variable "rds_connection_heartbeat" {
+  description = "If true, send a query to verify database connection is live on connection creation and retry on initial connection timeout.  Set to false if not using serverless RDS"
```

Comment:
Can we copy this updated description everywhere that the variable is defined?

Reply:
83548c67e